Composite/Core/WebClient/Media/ImageFormatProviders.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Linq;
+using Composite.Core.IO;
+using Microsoft.Extensions.DependencyInjection;
+
+
+namespace Composite.Core.WebClient.Media
+{
+    internal static class ImageFormatProviders
+    {
+        private static readonly ImageCodecInfo JpegCodecInfo = ImageCodecInfo.GetImageEncoders().First(codec => codec.FormatID == ImageFormat.Jpeg.Guid);
+
+        private static Action<Bitmap, string, int?> GetSaveInFormatFunction(ImageFormat imageFormat) =>
+            (resizedImage, outputFilePath, quality) => resizedImage.Save(outputFilePath, imageFormat);
+
+        internal static void AddDefaultImageFileFormatProviders(this IServiceCollection services)
+        {
+            var providers = new List<DefaultImageFileFormatProvider>
+            {
+                new DefaultImageFileFormatProvider(MimeTypeInfo.Jpeg, "jpeg", (resizedImage, outputFilePath, quality) =>

```

Comment:
Do we support  '.jpe', '.jif', '.jfif', '.jfi'?

Reply:
That extension is to be given to the "output" image, rather than being taken as an input. 
As input we accept "media type", f.e. "image/jpeg" instead of extension.
If an input image has ".jpe" extension, but it mapped into "image/jpeg" type - it will processed correctly.